### ISSUE TITLE
Could you update the README with an example with input proteins and ligands?

### ISSUE THREAD

**Issue by rubenalv** [phdymz/ProteinMAE#3]
Hello,
From the README, it is not clear how to supply a protein or a list thereof for training/inference. Could you clarify:
- how to supply inputs (proteins for --site, protein pairs for --search)
- the format of those inputs (pdb file, or processed tensor? If tensor, how to preprocess the -I suppose- pdb input?)

Thank you, the software looks promising!

**Comment by phdymz** [comment]
We used the method of modeling protein surface point cloud in dmasif to reconstruct the protein of ligand task in masif. It should be to reconstruct the pdb data into point cloud. You can try to use the point cloud generation code of dmasif to process your personal data.

**Comment by phdymz** [comment]
The environment version that dmasif (py37 ...) depends on is not consistent with my current code (we implement on py38 ...). It is difficult to plug its content directly and implement online calculation. Therefore, our current model still needs preprocessing and cannot process raw data. However, you can try to add a surface reconstruction module in dmasif yourself.

**Comment by rubenalv** [comment]
dMaSIF is a hard nut to crack, it took a lot of re-writing to make it work -that's why I got so excited when I found your paper! I'll reuse the pre-processing code from dMaSIF, thanks for the quick response.

**Comment by phdymz** [comment]
Yes, including masif, dmasif and masif versions are difficult to match. I don't have a good solution for this. I rented a computer with an older version before.
